React/Base/RCTRootViewInternal.h

```diff
@@ -26,7 +26,7 @@
  */
 #if TARGET_OS_TV
 @property (nonatomic, strong) RCTTVRemoteHandler *tvRemoteHandler;
-@property (nonatomic, strong) UIView *reactPreferredFocusedView;
+@property (nonatomic, weak) UIView *reactPreferredFocusedView;
```

Comment:
before this fix, UIView's reference was held even 30 seconds after unmount. I haven't tested anything above, assuming implication: 30s => will be there forever